src/main/java/uk/gov/hmcts/cmc/claimstore/documents/content/ClaimIssueReceiptContentProvider.java

```diff
@@ -0,0 +1,59 @@
+package uk.gov.hmcts.cmc.claimstore.documents.content;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.cmc.claimstore.services.staff.content.ClaimContentProvider;
+import uk.gov.hmcts.cmc.claimstore.services.staff.content.ClaimantContentProvider;
+import uk.gov.hmcts.cmc.claimstore.services.staff.content.PersonContentProvider;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+import uk.gov.hmcts.cmc.domain.models.otherparty.TheirDetails;
+import uk.gov.hmcts.cmc.domain.utils.PartyUtils;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static java.util.Objects.requireNonNull;
+
+@Component
+public class ClaimIssueReceiptContentProvider {
```

Comment:
The same here, this class and `SealedClaimContentProvider` has 99% in common, no point having two same implementations. Please merge together in form of having `ClaimContentProvider`

Reply:
This is till not done :(

Reply:
Im currently still doing it. The PR is still a WIP, i just wanted you to see if the current work ive done is okay before moving on